runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/SingletonDockerContainerManager.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.runners.fnexecution.environment;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.google.common.util.concurrent.SettableFuture;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.apache.beam.model.pipeline.v1.RunnerApi.Environment;
+import org.apache.beam.runners.fnexecution.GrpcFnServer;
+import org.apache.beam.runners.fnexecution.artifact.ArtifactRetrievalService;
+import org.apache.beam.runners.fnexecution.control.SdkHarnessClientControlService;
+import org.apache.beam.runners.fnexecution.logging.GrpcLoggingService;
+import org.apache.beam.runners.fnexecution.provisioning.StaticGrpcProvisionService;
+
+/**
+ * A {@link ContainerManager} which manages a single Docker Container.
+ *
+ * <p>The container is initialized on the first call to {@link #getEnvironment(Environment)}. Future
+ * calls will fail if there is more than one environment.
+ */
+public class SingletonDockerContainerManager implements ContainerManager {
```

Comment:
This class was also removed from the upstream PR. Please remove this class and all references to it. For now, just use a no-op ContainerManager and stub out where necessary in tests. We should have a replacement for it soon.